src/components/Leaderboard/Leaderboard.jsx

```diff
@@ -18,7 +17,7 @@ export class Leaderboard extends React.Component {
         return (
             <div className={styles.leaderboardPage}>
                 <div className={styles.backButton}>
-                    <Button data-cy="back" handleClick={this.goToLobby} text="Back" />
+                    <button onClick={this.goToLobby} data-cy="back" type='button'>Back</button>
```

Comment:
I don't think you actually need the `type` parameter if the button isn't in a form. I think that's just so that the form knows that the button won't submit.